ligo/src/token/fa2.mligo

```diff
@@ -0,0 +1,161 @@
+// SPDX-FileCopyrightText: 2020 TQ Tezos
+// SPDX-License-Identifier: LicenseRef-MIT-TQ
+
+// Corresponds to Token/FA2.hs module
+
+#include "../types.mligo"
+#include "../management.mligo"
+
+
+let unfrozen_token_id: nat = 0n
+let frozen_token_id: nat = 1n
+// let (|>) x f = f x
```

Comment:
I see you changed the `*_token_id` values, but you left behind this comment, let's remove it.